examples/logging.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use druid::shell::{runloop, WindowBuilder};
+use druid::widget::Column;
+use druid::{UiMain, UiState};
+use simple_logger;
+
+fn main() {
+    simple_logger::init().unwrap();
```

Comment:
I actually think we can get rid of this file, but I would like to add logging to a few examples; in particular `hello`, `calc`, and `custom_widget`. (These are the examples I mostly use when experimenting & debugging)

Reply:
Fair enough. ;) I've added logging to those examples.